manual/src/asciidoc/_06-gradle.adoc

```diff
@@ -25,6 +25,11 @@ tasks.apina {
     //  - 'STRING_UNION' => type MyEnum = "FOO" | "BAR" | "BAZ"
     enumMode = EnumMode.DEFAULT
 
+    // How nullables are translated to TypeScript interfaces? (Default mode is 'NULL'.)
+    //  - 'NULL'      => name: Type | null
+    //  - 'UNDEFINED' => name?: Type
```

Comment:
This is not what the implementation actually does, so either the documentation or the implementation needs to be changed.

Also, there's an unfortunate thing that these examples are duplicated in `README.md` so this needs to be added there as well. It feels too complex to setup some kind of system to automatically generate one from the other but no one reads the documentation so I've added the main bits to README as well.

Reply:
Oh right, I was actually initially pushing for this kind of change but then realized it may require a bit more changes and then implemented just the `.. | undefined` logic :)

Reply:
Actually I took another shot at it :) Now it should be correct! I also updated the README.